src/docs/css-helpers/animation.md

```diff
@@ -0,0 +1,21 @@
+# Animation
+
+The animation helper allows applying simple animations to UI elements.
+
+👉 Remember that non-block inline elements (ie. an unstyled `span` or elements
+that have `display: inline`) cannot be animated.
+
+<textarea is="docoff-react-preview">
+  <p>
+    <code className="mr-3">.animation-spin-clockwise</code>
+    <span className="d-inline-flex animation-spin-clockwise">
```

Comment:
Spinning icons are not centered.

Reply:
I managed to resolve it after all! I ROCK!